cwf/cloud/go/services/cwf/cwf/main.go

```diff
@@ -40,6 +42,14 @@ func main() {
 
 	builder_protos.RegisterMconfigBuilderServer(srv.GrpcServer, servicers.NewBuilderServicer())
 
+	specPath := config.GetSpecPath(cwf_service.ServiceName)
+	specServicer, err := swagger.NewSpecServicerWithPath(specPath)
+	if err != nil {
+		glog.Infof("Error retrieving Swagger Spec of service %s", cwf_service.ServiceName)
```

Comment:
Here and elsewhere
- No need to capitalize "Swagger Spec" -- just use "Swagger spec"
- This is indeed an error, so let's use `Errorf`
- Agreed though that it's not fatal, so you've handled that part correctly
- Also include the `%+v` of the error from the constructor wrapper
- Whenever we're swallowing an error, include a brief comment explaining why

Also, these 7 lines are basically the same across all main.go files. You could abstract out a `swagger.RegisterSpecServer(srv)` function to the swagger pkg and then your main.go changes would be one-liners

Reply:
Refactoring the changes doesn't actually seem to be that simple. My latest push includes changes which reduces the main.go changes to 2 lines(1 for intializing the servicer and 1 for registering it). Note that a key caveat of this change is that there is redundancy in the arguments passed to the servicer constructor namely the path to the spec file as well as the service itself. The "obvious" solution of solely passing in the service name actually generates a problem in the test suite which would require us to generate temporary directories of the form "/etc/magma/configs/swagger_spec" notably which does not exist. I opted to not pursue this solution for now as I feel like this would needlessly overpopulate the test image file system which I do not want to do. If anyone has any other solutions please let me know!

Reply:
My preference here would be to opt for a cleaner function that just takes service name rather than having to provide both the spec path and service name as these are somewhat redundant.

I'm okay with the slight pollution of the test image file system as long as this doesn't bleed into the production controller image. I don't see this being an issue though.

Reply:
Update: Seems like the parent directory of /etc/magma/orc8r/configs/ already existed so this was definitely the more concise solution. Thanks Michael